### DISCUSSION TITLE
Mode with more bad characters

### DISCUSSION THREAD

**Discussion by Rizhiy** [monkeytypegame/monkeytype#1225] [Ideas] [answered]
I can type some characters with greater accuracy than others. For practice, it would be nice to have a mode where difficult characters are more frequent.
I guess the easiest way to do that would be to keep track of accuracy for each character and then select words from the vocabulary which contain the most low-accuracy characters.

Ideally, you could sample words from the vocabularly where the weight for a word would be something like `prod(1/char_acc)`.
Alternatively, if vocabulary is small can just keep track of accuracy for each word.

EDIT: I guess this is similar to https://github.com/Miodec/monkeytype/discussions/1138, but for each character.

**Comment by Miodec** [comment, marked as answer]
This just got picked up by someone and should be in the works: #44
